docs/streams/getting-started-streams.md

```diff
@@ -0,0 +1,459 @@
+```
+
+</TabItem>
+</Tabs>  
+
+### Step 3. Create Global and Local Streams
+
+The streams in GDN can be either local or globally geo-replicated. The code below allows you to create either or both and then get the stream details.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+async function streams() {
+    try{
+      await console.log("Creating local stream...");
+      const stream_local = await client.createStream("testStream-local", true);
+
+      await console.log("Creating global stream...");
+      const stream_global = await client.createStream("testStream-global", false);
+
+    } catch(e){
+      await console.log("Streams could not be fetched because "+ e);
+    }
+}
+
+streams();
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+prefixText = ""
+prefixBool = False # If false, then the stream created below is global
+demo_stream = 'streamQuickstart'
+    
+# Get the right prefix for the streamName
+if prefixBool:
+    prefixText = "c8locals."
+else:
+    prefixText = "c8globals."
+
+# Create the stream if it doesn't already exist
+# To create both a global and local stream, run the code twice, once with prefixBool = True, once False
+streamName = {"stream-id": ""}
+if client.has_stream(demo_stream, local = prefixBool):
+    print("Stream already exists")
+    streamName["stream-id"] = concat(prefixText, demo_stream)
+    print ("OLD Producer =",  streamName["stream-id"])
+else:
+    streamName = client.create_stream(demo_stream, local=prefixBool)
+    print ("NEW Producer =",  streamName["stream-id"])
+
+# Get and print stream details
+print("Get streams: ", client.get_streams())
+```
+
+</TabItem>
+</Tabs>  
+
+### Step 4. Publish Messages
+
+Example to publish documents to a stream. The stream can be either a local stream or could be a geo-replicated stream.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+async function streams() {
+    try {
+
+      await console.log("Creating local stream...");
+      await client.createStream("my-stream", true);
+
+      const stream = await client.getStream("my-stream", true);
+      await stream.publishMessage("Hello World");
+
+      /*
+
+      // FOR ADVANCED USER
+      const stream = client.stream("my-stream", true);
+      await stream.createStream();
+
+      const producerOTP = await stream.getOtp();
+      const producer = await stream.producer("gdn.paas.macrometa.io", {
+        otp: producerOTP,
+    });
+      producer.on("open", () => {
+        // If you message is an object, convert the obj to string.
+        // e.g. const message = JSON.stringify({message:'Hello World'});
+        const message = "Hello World";
+        const payloadObj = { payload: Buffer.from(message).toString("base64") };
+        producer.send(JSON.stringify(payloadObj));
+      });
+
+      producer.on("message", (msg) => {
+        console.log(msg, "Sent Successfully");
+      });
```

Comment:
@diana-macrometa You will need this code `// FOR ADVANCED USER` there as well.

Reply:
Why do we need it? Is that what makes the later part work? 

Does the For Advanced User portion replace the earlier stuff? It looked like an alternate method of sending messages, so I thought it would be either/or.

Also, there were no errors with that portion when I removed it.

Reply:
so if you see the above part is not required as we are doing the same steps except publishing line. but for producers, you will have to use an advanced part.